force-app/main/default/pages/CON_ContactMerge.page

```diff
@@ -164,7 +164,7 @@
                                         <th class="slds-text-heading_label" scope="col">Duplicate Record Set Name</th>
                                         <th class="slds-text-heading_label" scope="col">Last Modified Date</th>
                                         <th class="slds-text-heading_label" scope="col">Duplicate Rule</th>
-                                        <th class="slds-text-heading_label" scope="col">Record Count</th>
+                                        <th class="slds-text-heading_label" scope="col">{!$Label.commonContactCount}</th>
```

Comment:
It looks like this page has a lot of hard-coded text (wow). Not that it all has to be fixed here, but can you open a new localization WI to fix the rest of the labels (if you haven't done that already)?

Reply:
Labels very nicely handled!